shadows/framework/src/main/java/org/robolectric/shadows/ShadowContentResolver.java

```diff
@@ -530,6 +531,20 @@ protected static boolean isSyncActive(Account account, String authority) {
     return status != null && status.syncRequests > 0;
   }
 
+  @Implementation
+  protected static List<SyncInfo> getCurrentSyncs() {
```

Comment:
Please add test for this shadow implementation. We need every PR adds related tests for its Robolectric's implementation changes.

Reply:
👍 on a test, in ShadowContenResolverTest. It can be something simple like having a call to ContentResolver.requestSync and verifying that the result is in ContentResolver.getCurrentSyncs.

Also maybe checking that if no syncs get requested, ContentResolver.getCurrentSyncs returns the empty list.

Reply:
Also a test that calls requestSync followed by cancelSync, and the value should not be in `ContentResolver.getCurrentSync`.

Reply:
ok thanks, i got it.
i implement it.

Reply:
`
@Test

  public void shouldGetCurrentSyncs() {

    assertThat(ContentResolver.getCurrentSyncs().size()).isEqualTo(0);
    ContentResolver.requestSync(a, AUTHORITY, new Bundle());
    ContentResolver.requestSync(b, AUTHORITY, new Bundle());

    List<SyncInfo> syncs = ContentResolver.getCurrentSyncs();
    assertThat(syncs.size()).isEqualTo(2);

    SyncInfo first = syncs.get(0);
    assertThat(first.account).isEqualTo(a);
    assertThat(first.authority).isEqualTo(AUTHORITY);

    SyncInfo second = syncs.get(1);
    assertThat(second.account).isEqualTo(b);
    assertThat(second.authority).isEqualTo(AUTHORITY);

    ContentResolver.cancelSync(a, AUTHORITY);
    List<SyncInfo> syncsAgain = ContentResolver.getCurrentSyncs();
    assertThat(syncsAgain.size()).isEqualTo(1);

    SyncInfo firstAgain = syncsAgain.get(0);
    assertThat(firstAgain.account).isEqualTo(b);
    assertThat(firstAgain.authority).isEqualTo(AUTHORITY);
  }`

This is the test i have added in ShadowContentResolverTest.java